app/assets/stylesheets/find/components/_save-course-button.scss

```diff
@@ -5,6 +5,27 @@
 
   background: none;
   border: none;
+
+  &:focus {
+    outline: none;
+    outline-color: $govuk-focus-colour;
+
+    background: $govuk-focus-colour;
+    border: none;
+    border-bottom: 3px solid govuk-colour("black");
+  }
+
+  &:focus .save-course-button__text {
+    color: govuk-colour("black");
+  }
+}
+
+.results-save-course-button__unstyled-button {
+  float: right;
+
+  @include govuk-media-query($until: tablet) {
+    float: left;
+  }
```

Comment:
I am not sure I fully understand the CSS changes to this PR that send events to Bigquery. Was there a reason?